docs/dev/tutorials/cross-chain-tutorial.md

```diff
@@ -293,47 +388,61 @@ export default async function (hre: HardhatRuntimeEnvironment) {
 }
 ```
 
-7. After replacing `<WALLET-PRIVATE-KEY>` and `<GOVERNANCE-ADDRESS>` with the private key of an Ethereum wallet with some ETH balance on Göerli, and the address of the L1 governance contract respectively, run the script using the following command:
+2. Now deploy the contract from the `L2-counter/` folder root to zkSync:
 
+::: code-tabs
+@tab npm
+```sh
+npx hardhat deploy-zksync
 ```
+@tab yarn
+```sh
 yarn hardhat deploy-zksync
 ```
+:::
 
-In the output, you should see the address to which the contract was deployed.
-
-::: tip
+You should see output like this:
 
-You can find more specific details about deploying contracts in the [quickstart tutorial](../building-on-zksync/hello-world.md) or the documentation for the [hardhat plugins](../../api/hardhat/getting-started.md) for zkSync.
+```txt
+Running deploy script for the Counter contract
+Counter was deployed to 0x3c5A6AB2390F6217C78d2F6F403A9dFb7e7784FC
+```
 
+:::warning
+At this point, if you have been following the `yarn` flow and you're on a Mac M1, you may have noticed `gyp` errors in the output. These may hinder your progress. If so, to bypass them, do the following from the `L2-counter` directory:
+* Remove `node_modules/`.
+* Open `package.json` and change the version of `matterlabs/hardhat-zksync-solc` to `^0.3.15-beta.2`.
+* Delete the `artifacts` and `caches` folders.
+* Run `yarn`.
```

Comment:
Also need to mention, in `hardhat.config.ts`, change zksolc from 1.3.5 to 1.3.6:

```
module.exports = {
  zksolc: {
    version: "1.3.1",
    version: "1.3.6",
    ```